server.js

```diff
@@ -152,25 +177,51 @@ app.get('/broad', (req, res) => {
 
 // LOGIN RELATED INFORMATION
 
-app.get('/isLoggedIn', (req, res) => {
-  res.send('hitting server!!!!');
-});
-
 app.post('/login', (req, res) => {
-  // TODO: Senai do some stuff here
-  console.log('server post login endpoint');
-  console.log(req.body, 'rrreeeqqqq......bbbbooooddddyyyy');
-  // helpers.loginUser(req, res);
-  // helpers.createUser(req, res);
-  res.send(req.body);
-  // res.send('logged in');
+  const userInfo = req.body;
+  console.log(userInfo, '88888888888888888');
```

Comment:
long console logs. again still in progress?

Reply:
Nope, not in progress, I just forgot to remove it. Cleaning it up now.